src/controllers/action-queue/types/pushTypes.js

```diff
@@ -0,0 +1,142 @@
+// @flow
+
+/**
+ * Firebase admin credentials.
+ *
+ * We should probably match the `ServiceAccount` type from the Firebase SDK,
+ * but this is the existing situation in the database.
+ */
+export type FirebaseAdminKey = {
+  type?: string,
+  project_id?: string,
+
+  auth_provider_x509_cert_url?: string,
+  auth_uri?: string,
+  client_email?: string,
+  client_id?: string,
+  client_x509_cert_url?: string,
+  private_key_id?: string,
+  private_key?: string,
+  token_uri?: string
+}
+
+/**
+ * An API key, along with some extra info.
+ */
+export type ApiKey = {
+  apiKey: string,
+  appId: string,
+
+  admin: boolean,
+  adminsdk?: FirebaseAdminKey
+}
+
+/**
+ * An app installed on a single phone.
+ *
+ * This the in-memory format, independent of the database.
+ */
+export type Device = {
+  +created: Date,
+  +deviceId: string,
```

Comment:
These first three (api key & device) are all internal types of the server you cannot access, and should just delete. The rest of the file has valid types that go with the cleaners.

Reply:
Removed the three types: `FirebaseAdminKey`, `ApiKey`, and `Device`.